cgmes/cgmes-conformity/src/test/java/com/powsybl/cgmes/conformity/test/CgmesConformity1NetworkCatalog.java

```diff
@@ -1028,6 +1034,42 @@ public static Network microType4BE() {
                 .setP0(-11.518776)
                 .setQ0(67.377544);
 
+        network.getShuntCompensator("_002b0a40-3957-46db-b84a-30420083558f").remove();
+        network.getVoltageLevel("_469df5f7-058f-4451-a998-57a48e8a56fe")
+                .newShuntCompensator()
+                    .setId("_002b0a40-3957-46db-b84a-30420083558f")
+                    .setName("BE_S2")
+                    .setConnectableBus(busBrussels380.getId())
+                    .setBus(busBrussels380.getId())
+                    .setCurrentSectionCount(1)
+                    .newNonLinearModel()
+                        .beginSection()
+                            .setSectionIndex(1)
```

Comment:
Is it really need to specify an index? It could be implicit (builder order)

Reply:
cf answer below